src/NJsonSchema/Annotations/JsonSchemaPostProcessAttribute.cs

```diff
@@ -0,0 +1,35 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="JsonSchemaAttribute.cs" company="NJsonSchema">
+//     Copyright (c) Rico Suter. All rights reserved.
+// </copyright>0
+// <license>https://github.com/rsuter/NJsonSchema/blob/master/LICENSE.md</license>
+// <author>Rico Suter, mail@rsuter.com</author>
+//-----------------------------------------------------------------------
+namespace NJsonSchema.Annotations
+{
+    using System;
+
+    /// <summary>
+    /// Allows to post process schema.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Property, AllowMultiple = true)]

```

Comment:
Attribute usage should only be class, otherwise multiple processors per type could be registered which may lead to strange behaviors